ui/src/app/shared/components/track-user-interface-event.tsx

```diff
@@ -0,0 +1,10 @@
+import {services} from '../../shared/services';
+import { EventParam } from '../services/info-service';
+
+export const TrackEvent = ({name}: {name: string}): null => {
```

Comment:
minor - no need to specify null,

Reply:
https://github.com/argoproj/argo-workflows/runs/6196069776?check_suite_focus=true#step:5:1333
if I remove this, error occurs like this.